src/lib/messenger.js

```diff
@@ -38,6 +42,10 @@ export default function messenger(bot) {
         return strippedText;
       }
 
+      function isDefined(variable) {
```

Comment:
It's actually sufficient to check directly like this:

```js
...
else if (newChatMember) {
  // handle this
} else if (leftChatMember) {
  // handle that
}
...
```

because a variable evaluates to `true` if it is set, and if it is set to `undefined` or `null` they are interpreted as `false` see [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) and [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) values.